### PR TITLE
add 3d interactivity docs

### PR DESCRIPTION
**Pull request by kevinyamauchi**
# Description
This PR adds a 3D interactivity guide. This was approved in https://github.com/napari/napari.github.io/pull/144, but it should have been merged here instead of on the website repo 🤦 

## Type of change
- [x] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: docs/_toc.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 - file: guides/index
   sections:
+    - file: guides/3d_interactivity
```
Comment (kevinyamauchi) [inline docs/_toc.yml]:
```suggestion
    - file: guides/3D_interactivity
```

**Review by kevinyamauchi — COMMENTED** [review]
Ah, I had the file name wrong.

**Comment by alisterburt** [comment]
We want this in with the upcoming release candidate I think - given that, and the fact it was already reviewed, I will merge tomorrow morning if there are no issues raised 🙂

**Review by sofroniewn — APPROVED** [review]
LGTM! I'll merge now so I can make the first RC today!!

**Pull request merged**